ko/easy-concat.md

```diff
@@ -0,0 +1,55 @@
+---
+id: 533
+title: Concat
+lang: ko
+level: easy
+tags: array
+---
+
+## 챌린지
+
+타입 시스템 내에서 자바스크립트의 `Array.cocat` 함수를 구현해보세요.
```

Comment:
Typo?

```suggestion
타입 시스템 내에서 자바스크립트의 `Array.concat` 함수를 구현해보세요.
```

Reply:
My mistake...😢

Reply:
@ghaiklor  I missed this suggestion. Could you please review it again?

Reply:
Done, thanks!